src/manage-fonts/index.js

```diff
@@ -228,30 +232,36 @@ export default function () {
 					) }
 				</h1>
 				<p>
-					{ __(
-						'You can manage your theme fonts directly from the',
-						'create-block-theme'
-					) }{ ' ' }
-					<a href="/wp-admin/site-editor.php?canvas=edit">
-						{ _x(
-							'editor',
-							'Link to the site editor',
+					{ createInterpolateElement(
+						__(
+							'You can manage your theme fonts directly from the <a>editor</a>.',
 							'create-block-theme'
-						) }
-					</a>
-					.
+						),
+						{
+							a: (
+								// eslint-disable-next-line jsx-a11y/anchor-has-content
+								<a href="/wp-admin/site-editor.php?canvas=edit" />
```

Comment:
Using an absolute path like this will break when using non-standard hosting configurations, like putting core files in a subdirectory, or subdirectory multisite installations.

But that's not a new problem with this PR.

Reply:
Thanks, I think we should handle the link structure change separately from this PR.